tools/perf/ResultsComparer/Program.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Xml;
+using Perfolizer.Mathematics.Multimodality;
+using Perfolizer.Mathematics.SignificanceTesting;
+using Perfolizer.Mathematics.Thresholds;
+using CommandLine;
+using MarkdownLog;
+using Newtonsoft.Json;
+using ResultsComparer.Core;
+using ResultsComparer.Bdn;
+using System.Threading.Tasks;
+
+namespace ResultsComparer
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // we print a lot of numbers here and we want to make it always in invariant way
+            Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
+
+            Parser.Default.ParseArguments<CommandLineOptions>(args).WithParsed((options) => Compare(options).Wait());
+        }
+
+        private static async Task Compare(CommandLineOptions args)
+        {
+            IResultsComparer comparer = new BdnComparer();
+
+            try
+            {
+                ComparerOptions options = new ComparerOptions
+                {
+                    StatisticalTestThreshold = args.StatisticalTestThreshold,
+                    NoiseThreshold = args.NoiseThreshold,
+                    FullId = args.FullId,
+                    TopCount = args.TopCount,
+                    Filters = args.Filters
+                };
+
+                var results = await comparer.CompareResults(args.BasePath, args.DiffPath, options);
+
+                if (results.NoDiff)
+                {
+                    return;
+                }
+
+                var resultsArray = results.Results.ToArray();
+                PrintSummary(resultsArray);
+
+                PrintTable(resultsArray, EquivalenceTestConclusion.Slower, args);
+                PrintTable(resultsArray, EquivalenceTestConclusion.Faster, args);
+
+
+            }
+            catch (Exception ex)
+            {
+                Console.Error.WriteLine($"Error: {ex.Message}");
+            }
+        }
+        private static void PrintSummary(ComparerResult[] notSame)
+        {
+            var better = notSame.Where(result => result.Conclusion == EquivalenceTestConclusion.Faster);

```

Comment:
```suggestion
            var better = notSame.Where(result => result.Conclusion == EquivalenceTestConclusion.Faster).ToArray();

```
Same for the one below.